src/pudl/etl/cli.py

```diff
@@ -158,12 +148,13 @@ def pudl_etl(
         },
     }
 
+    # Limit the number of concurrent workers when launch assets that use a lot of memory.
     tag_concurrency_limits = [
         {
-            "key": "datasource",
-            "value": "epacems",
-            "limit": epacems_workers,
-        }
+            "key": "memory-use",
+            "value": "high",
+            "limit": 2,
+        },
```

Comment:
So this means no more than two high-memory processes launched by dagster will be running at the same time.  Hopefully this is enough to keep our nightly builds chugging along. If it doesn't work we can limit the concurrency globally and/or add the tag to more high memory assets.

Reply:
Maybe we should add some tags to other high memory assets that were running when the VM failed?

```
_core_eia860__boilers
_core_eia860__ownership
_core_eia860__plants
_core_eia860__utilities
_core_eia923__boiler_fuel
_core_eia923__coalmine
_core_eia923__cooling_system_information
_core_eia923__fgd_operation_maintenance
_out_ferc1__yearly_steam_plants_sched402_with_plant_ids.ferc_to_ferc.merge_steam_fuel_dfs
_out_ferc714__hourly_demand_matrix
core_demand_side_management_eia861
core_eia861__yearly_balancing_authority
core_eia861__yearly_dynamic_pricing
extract_phmsagas
raw_eia930__balance
raw_eia930__interchange
raw_eia930__subregion
```

Given CEMS and `_out_ferc714__hourly_demand_matrix` are the only tagged assets it's unlikely they will be executed at the same time and limit concurrency.